markdown/xliffs/sample-md-nl-NL.xliff

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2">
+  <file original="md/en-US/README.md" source-language="en-US" target-language="nl-NL" product-name="sample-md">
+    <body>
+      <trans-unit id="1" resname="Title" restype="string" datatype="x-yaml">
+        <source>Readme for the Github-Flavored Markdown Sample</source>
+        <target>Readme voor het Github-flavored markdown voorbeeld</target>
+      </trans-unit>
+    </body>
+  </file>
+  <file original="md/en-US/README.md" source-language="en-US" target-language="nl-NL" product-name="sample-md">
```

Comment:
Is there a specific reason to put into two file tags? All data is from the same file. I think it should be in the same file tag. isn't it?

Reply:
I merged them. Originally there was a bug in ghfm where the "original" tag was 2 different things for the frontmatter and the body of the markdown. When I fixed that, I updated the original and didn't think about making the two "file" tags into one.